crawler.js

```diff
@@ -11,7 +11,7 @@ const login = async (browser, config) => {
 
   await page.type(selectors.login.username, credentials.username);
   await page.type(selectors.login.password, credentials.password);
-  await page.click(selectors.login.submit);
+  page.click(selectors.login.submit);
```

Comment:
Interesting. I was actually going to implement something like `clickAndWaitForNavigation ` mentioned [here](https://github.com/GoogleChrome/puppeteer/issues/1412#issuecomment-345509476), but decided to go against it.

Can you combine it and use this instead? It makes more sense since [page.click returns a Promise](https://pptr.dev/#?product=Puppeteer&version=v1.13.0&show=api-pageclickselector-options):
```js
await Promise.all([
  page.waitForNavigation() 
  page.click('a')
]);
```